FLT/Deformations/LiftFunctor.lean

```diff
@@ -69,4 +91,53 @@ def deformationFunctor (ρ : (repnFunctor n G 𝓞).obj .residueField) :
     Subfunctor (repnQuotFunctor n G 𝓞) :=
   .ofIsTerminal _ ProartinianCat.isTerminalResidueField {(toRepnQuot n G 𝓞).app _ ρ}
 
+/-- The subfunctor of flat lifts. This probably only makes sense when `𝓞` is `v`-adic. -/
+def flatFunctor (v : Ω K) : Subfunctor (repnFunctor n (Γ K) 𝓞) where
+  obj R := { ρ | (toFramedGaloisRep ρ).IsFlatAt v }
+  map := sorry -- See e.g. Conrad Theorem 1.6 of CSS
+
+/-- The subfunctor of unramified (at `v`) representations. -/
+def unramifiedFunctor (v : Ω K) : Subfunctor (repnFunctor n (Γ K) 𝓞) where
```

Comment:
I'm surprised that you don't do "unramified outside S" -- is the idea to just use infinitely many of these? Does that cause problems?